### PR TITLE
NEPT-1648: Update ecl-builder workflow with up to date dependencies.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: ecl-builder.config.js [hunk 1]
```diff
@@ -13,19 +13,9 @@ module.exports = {
...
-      ),
-      dest: path.resolve(__dirname, "assets/styles/europa.css"),
-      options: {
-        sourceMap: false
-      }
-    },
     {
       entry: path.resolve(__dirname, "assets/src/scss/editor-index.scss"),
-      dest: path.resolve(__dirname, "assets/styles/wysiwyg/editor.css"),
+      dest: path.resolve(__dirname, "assets/styles/editor.css"),
```
Comment (voidtek) [inline ecl-builder.config.js]: this change does not need to be in this ticket.

File: ecl-builder.config.js [hunk 2]
```diff
@@ -13,19 +13,9 @@ module.exports = {
     }
   ],
   styles: [
-    {
-      entry: path.resolve(
-        source_folder,
-        "ecl-components-preset-base/index.scss"
```
Comment (voidtek) [inline ecl-builder.config.js]: ecl-components-preset-base should be ecl-components-preset-full, no ?

Comment (drupol) [reply ecl-builder.config.js]: drupol> @yannick help with https://www.npmjs.com/package/@ec-europa/ecl-components-preset-base
<yannick> @drupol this one is deprecated, please use https://www.npmjs.com/package/@ec-europa/ecl-preset-full

File: ecl-builder.config.js (hunk 1 above)
Comment (drupol) [reply ecl-builder.config.js]: Fixed, thanks.

**Pull request merged**